setup.cfg

```diff
@@ -38,6 +38,8 @@ install_requires =
   numpy
   matplotlib
   scipy
+  cdflib>=0.3.20
```

Comment:
How many sources use this?

I would prefer this being an optional dependency in some manner.

Reply:
One so far PSP/RFS but will also be needed for SO/RPW but can make it optional and I guess and `try / except` to the the imports?

Reply:
Hmm, if you plan to add more that need it, maybe its not worth it in the long run.